awx/ui_next/src/api/models/Credentials.js

```diff
@@ -16,8 +16,11 @@ class Credentials extends Base {
   }
 
   readInputSources(id) {
+    const maxRequests = 5;
```

Comment:
At first this function was a bit confusing, but after looking at it again I like it.